pkg/kubernetes/podhelper/container_logs_test.go

```diff
@@ -0,0 +1,96 @@
+package podhelper
+
+import (
+	"testing"
+
+	"k8s.io/client-go/kubernetes/fake"
+
+	tassert "github.com/stretchr/testify/assert"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestHasNoBadLogsPodHasContainer(t *testing.T) {
```

Comment:
not sure from the name which function/check this test is specifically for. May help to rename for clarity since this seems similar to TestPodHasContainer?

Reply:
I've renamed it. This test ensures that a failed check is raised if the init container isn't present. I'll be adding additional test cases for this test (such as testing whether the regex search works) in a future PR.

Reply:
oh great, could eventually be useful for testing HasOsmInitContainerCheck and HasEnvoyImageCheck too!!